backend/lobby/consumers_lobby_test.py

```diff
@@ -0,0 +1,108 @@
+# import pytest
```

Comment:
Is there a reason for a completely commented out test file?

Reply:
these tests work locally, but for some reason (I've not determined) these Django tests choke on the server.

Guess I can remove them though.

Reply:
I don't think tests that fail on the server should just be completely deleted, they still have value run locally. Instead of deleting you can just add an ignore argument to the Travis.yml py.test command. This way we still get the benefit of having the tests available but don't bother with the Django tests on Travis